Web/Scotty/Internal/Types.hs

```diff
@@ -231,10 +239,11 @@ instance (MonadUnliftIO m) => MonadPlus (ActionT m) where
 
 -- | catches either ActionError (thrown by 'next') or 'StatusError' (thrown if e.g. a query parameter is not found)
```

Comment:
the docstring should be updated to show that we also catch 'ScottyException' here

Reply:
done